test/lightning/invocation_test.exs

```diff
@@ -10,6 +11,8 @@ defmodule Lightning.InvocationTest do
 
   require SearchParams
 
+  setup :verify_on_exit!
```

Comment:
@elias-ba Can you please put me out of my misery and explain why this is here :) ?

Reply:
Thanks for the catch. I initially tried to mock the `export_workorders` func. Then forgot to remove that line